README.md

```diff
@@ -268,7 +280,8 @@ As always you need to navigate to the root folder of this repository first. Make
     --restart always \
     --name eth-duties \
     tobiwo/eth-duties:latest \
-    --validators 123456 456789 \
+    --validators 123456,456789 \
+    --validators "111;My_Validator","222;Validator2" \
```

Comment:
either choose comma or space separation. or should this show that both ways are possible?